embedded-graphics/src/primitives/arc/styled.rs

```diff
@@ -102,6 +103,18 @@ where
     }
 }
 
+impl<C> Dimensions for Styled<Arc, PrimitiveStyle<C>>
+where
+    C: PixelColor,
+{
+    // FIXME: This doesn't take into account start/end angles. This should be fixed to close #405.
+    fn bounding_box(&self) -> Rectangle {
```

Comment:
This would also be the same for all the other primitives, no? If they're transparent, the bounding box would be zero sized. For consistency, should that be represented as `Rectangle::new(Point::zero(), Size::zero())` or `Rectangle::new(primitive_top_left, Size::zero())` in all cases?

Reply:
> This would also be the same for all the other primitives, no?

Yes, I forgot to consider completely transparent primitives. `Arc` and `Line` are special because they don't use the fill.

> For consistency, should that be represented as `Rectangle::new(Point::zero(), Size::zero())` or `Rectangle::new(primitive_top_left, Size::zero())` in all cases?

I would prefer to have the point be inside the primitive. Normally the `top_left` shouldn't matter for a zero sized rectangle, but I encountered some problems with this some time ago, but I cannot remember the details at the moment. How about `Rectangle::new(primitive.bounding_box().center(), Size::zero())`?

Reply:
Sounds good. I'll add that in.